main/signers/hot/SeedSigner/index.js

```diff
@@ -27,7 +27,7 @@ class SeedSigner extends HotSigner {
       const addresses = []
       for (let i = 0; i < 100; i++) {
         const publicKey = wallet.derive('m/44\'/60\'/0\'/0/' + i).publicKey
-        const address = publicKeyToAddress(publicKey)
+        const address = computeAddress(publicKey)
```

Comment:
need to make sure to test this change thoroughly as this is a very sensitive section of the code. @floating just raising this in case you have additional insight here